docs/utils.rst

```diff
@@ -4,65 +4,36 @@ DeepChem has a broad collection of utility functions. Many of these
 maybe be of independent interest to users since they deal with some
 tricky aspects of processing scientific datatypes.
 
-Array Utilities
----------------
-
-.. autofunction:: deepchem.utils.pad_array
-
-Data Directory
```

Comment:
I think people use `dc.utils.get_data_dir()` to check where deepchem data is being stored so we should add this back in to the docs

Reply:
Ah, I see this is `dc.utils.data_utils.get_data_dir()` now. I think it would still be good to keep a separate section in the docs discussing the data directory as something users should know about